MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -23,7 +23,9 @@ open class NavigationRouteOptions: RouteOptions {
         includesSteps = true
         routeShapeResolution = .full
         attributeOptions = [.congestionLevel, .expectedTravelTime]
+        includesVoiceInstructions = true
         includesExitRoundaboutManeuver = true
+        distanceMeasurementSystem = .imperial
```

Comment:
Pull this logic out into an extension on Locale, then get the current locale’s measurement system:

https://github.com/mapbox/mapbox-navigation-ios/blob/e1b588b74e9be38b8ba207cf39a553532d5bb650/MapboxCoreNavigation/DistanceFormatter.swift#L31-L37